src/user-management/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>User Management</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1>User Management</h1>
+    </header>
+    <section id="main-section">
+      <div id="search">
+        <input id="user-search" type="text" placeholder="Search Users" />
+      </div>
+      <div id="view-type-btn">
+        <button id="tile-view-btn">
+          <img
+            id="icon-first"
+            src="./images/hamburger.png"
+            width="25"
+            height="25"
+          />
+        </button>
+        <button id="table-view-btn" class="btn-active">
+          <img
+            id="icon-second"
+            src="./images/four-squares-button.png"
+            width="25"
+            height="25"
+          />
+        </button>
+      </div>
+      <div id="user-list">
+        <div id="loader"></div>
+      </div>
+    </section>
+  </body>
+  <script src="constants.js"></script>
+  <script src="utils.js"></script>
+  <script src="script.js"></script>
```

Comment:
Why do these scripts not use `defer` like earlier? We should try to maintain consistency in our usage patterns.